server/athenian/api/controllers/settings_controller.py

```diff
@@ -462,9 +462,15 @@ async def set_logical_repository(request: AthenianWebRequest, body: dict) -> web
                             web_model.name, full_name, prefixed_name, repo_id,
                             web_model.account, sdb_conn, request.pdb)
                     break
+            else:
+                pass
+                # TODO: what happens here ?? no deletion and there's a following insert
```

Comment:
other open issue

Reply:
done, please check if release settings match comparison is right

Reply:
It's fine :+1: